### ISSUE TITLE
EME: Range control doesn't have a label

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#241]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/eme/

The range control for the bitrate doesn't have a label associated with it. The label is a P element in the DIV above. 

Should be changed to a label element with a for="bitrateSlider". 

Another thing to note is the value should be kept in sync with the bitrate value after it. I'm skeptical this is the case as it has a max="7" applied to it, while bitrate goes up to 59999

**Comment by alrra** [comment]
Closing in favor of #253.
